### PR TITLE
Add commands to populate the database with data

### PR DESCRIPTION
**Pull request by MaiMee1**
New commands:

 - `loadrecipe`
 - `loadingr`

+ A script for getting the nutrients of an ingredient by using FoodData Central's search API.

***NEW!!!***

- `scrape <source> <num>`

Script to scrape useful data ourselves (comes with everything including images)  
Currently supports only allrecipes: `scrape allrecipes 10`, e.g.

**UPDATE:** added support for epicurious

### CODE REVIEW

File: koocook_core/management/commands/_add_path.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def add_datatrans():
+    import sys
+    sys.path.insert(0, 'C:\\Users\\User\\PycharmProjects\\datatrans')
```
Comment (th-bunratta) [inline koocook_core/management/commands/_add_path.py]: This takes the local files and depends on a particular machine. Some changes, perhaps?

Comment (MaiMee1) [reply koocook_core/management/commands/_add_path.py]: Resolved in a24169

**Comment by MaiMee1** [comment]
Why force push.... Now conflicts are harder to solve.

File: koocook_core/models/recipe.py [hunk 2]
```diff
@@ -46,6 +46,19 @@ def view_count(self) -> int:
...
+        if save:
+            self.save()
+
+    @property
+    def popularity_score(self) -> float:
+        return ((float(self.view_count)*1.9)+(float(self.aggregate_rating.rating_value)*3.1))/5
+
     @property
     def popularity_score(self) -> float:
         return ((float(self.view_count)*1.9)+(float(self.aggregate_rating.rating_value)*3.1))/5
```
Comment (th-bunratta) [inline koocook_core/models/recipe.py]: Duplicate lines of code. Remove?

**Pull request merged**